src/Catalyst.Core.Lib.Tests/UnitTests/Validators/TransactionValidatorTests.cs

```diff
@@ -166,6 +167,8 @@ public bool Verify(ISignature signature, ReadOnlySpan<byte> message, ReadOnlySpa
             {
                 return _verifyResult;
             }
+
+            public bool BatchVerify(IList<ISignature> signatures, IList<byte[]> messages, ReadOnlySpan<byte> context) { throw new NotImplementedException(); }
```

Comment:
is this correct?

Reply:
Same as below, batch verification isn't needed for these tests